### PR TITLE
update docs labels

### LINKED ISSUE #34115: Update docs to use recommended labels
**This is a Feature Request**
Requesting that docs be updated to use the Kubernetes [recommended labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/) for service objects

**What would you like to be added**

Change app: MyApp examples to app.kubernetes.io/name: MyApp

**Why is this needed**

This caused some confusion for our engineers of what the best labels are to use and these docs examples often get copy pasted, having them be aligned to recommended labels makes sense.

**Comments**

N/A

### PR DESCRIPTION
**Pull request by TheRealDwright**
Updates docs to use kubernetes recommended labels
as outlined in the documentation.

- [en] update en docs to use recommended labels

Closes #34115

### CODE REVIEW

**Comment by bradtopol** [comment]
Assigning @kbarnard10 since this affects so much blog content
/assign @kbarnard10

**Review by joshdk — APPROVED** [review]
This looks great!

**Comment by Sea-n** [comment]
Hey @TheRealDwright,

Please limit this PR to only change `content/en`, and open different PR for each language (or leave it to other contributor).

Feel free to `/unhold` yourself once it's done.
/hold

**Comment by Sea-n** [comment]
Thanks! Then I can help with adjusting label.

/remove-language es fr id ja ko pt uk zh

File: content/en/blog/_posts/2018-07-18-11-ways-not-to-get-hacked.md [hunk 1]
```diff
@@ -186,7 +186,7 @@ metadata:
   name: my-service
 spec:
   selector:
-    app: MyApp
+    app.kubernetes.io/name: MyApp
```
Comment (sftim) [inline content/en/blog/_posts/2018-07-18-11-ways-not-to-get-hacked.md:189]: We avoid changing blog articles that are more than 1 year old.

File: content/en/blog/_posts/2019-02-12-building-a-kubernetes-edge-control-plane-for-envoy-v2.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ metadata:
         service: my-service
 spec:
   selector:
-    app: MyApp
+    app.kubernetes.io/name: MyApp
```
Comment (sftim) [inline content/en/blog/_posts/2019-02-12-building-a-kubernetes-edge-control-plane-for-envoy-v2.md:41]: We avoid changing blog articles that are more than 1 year old.

**Review by sftim — CHANGES_REQUESTED** [review]
I'd prefer _not_ to revise the old blog articles. The other changes seem OK to me (I haven't checked thoroughly).

File: content/en/blog/_posts/2018-07-18-11-ways-not-to-get-hacked.md (hunk 1 above)
Comment (TheRealDwright) [reply content/en/blog/_posts/2018-07-18-11-ways-not-to-get-hacked.md]: Please note the consistency in this file is an issue, L189 `app: MyApp` vs L229 `app: myapp`

Have removed the changes regardless.

File: content/en/docs/concepts/services-networking/dual-stack.md [hunk 3]
```diff
@@ -93,9 +93,13 @@ set the `.spec.ipFamilyPolicy` field to one of the following values:
...
     of the first element in the `.spec.ipFamilies` array.
 
+<<<<<<< HEAD
 If you would like to define which IP family to use for single stack or define the order of IP
 families for dual-stack, you can choose the address families by setting an optional field,
-`.spec.ipFamilies`, on the Service. 
+`.spec.ipFamilies`, on the Service.
+=======
+If you would like to define which IP family to use for single stack or define the order of IP families for dual-stack, you can choose the address families by setting an optional field, `.spec.ipFamilies`, on the Service.
+>>>>>>> cca4c78633 ([en] update en docs to use recommended labels)
```
Comment (sftim) [inline content/en/docs/concepts/services-networking/dual-stack.md:102]: This looks like a manual merge gone wrong; please fix it.

File: content/en/docs/concepts/services-networking/dual-stack.md [hunk 4]
```diff
@@ -118,6 +122,7 @@ These examples demonstrate the behavior of various dual-stack Service configurat
 
 #### Dual-stack options on new Services
 
+<<<<<<< HEAD
```
Comment (sftim) [inline content/en/docs/concepts/services-networking/dual-stack.md:125]: This looks like a manual merge gone wrong; please fix it.

File: content/en/docs/concepts/services-networking/dual-stack.md [hunk 5]
```diff
@@ -133,13 +138,24 @@ These examples demonstrate the behavior of various dual-stack Service configurat
...
+
+{{< codenew file="service/networking/dual-stack-default-svc.yaml" >}}
+
+1. This Service specification explicitly defines `PreferDualStack` in `.spec.ipFamilyPolicy`. When you create this Service on a dual-stack cluster, Kubernetes assigns both IPv4 and IPv6 addresses for the service. The control plane updates the `.spec` for the Service to record the IP address assignments. The field `.spec.ClusterIPs` is the primary field, and contains both assigned IP addresses; `.spec.ClusterIP` is a secondary field with its value calculated from `.spec.ClusterIPs`.
+
+      * For the `.spec.ClusterIP` field, the control plane records the IP address that is from the same address family as the first service cluster IP range.
+      * On a single-stack cluster, the `.spec.ClusterIPs` and `.spec.ClusterIP` fields both only list one address.
+      * On a cluster with dual-stack enabled, specifying `RequireDualStack` in `.spec.ipFamilyPolicy` behaves the same as `PreferDualStack`.
+>>>>>>> cca4c78633 ([en] update en docs to use recommended labels)
```
Comment (sftim) [inline content/en/docs/concepts/services-networking/dual-stack.md:158]: This looks like a manual merge gone wrong; please fix it.

File: content/en/docs/concepts/services-networking/dual-stack.md [hunk 6]
```diff
@@ -165,6 +181,7 @@ dual-stack.)
 
    You can validate this behavior by using kubectl to inspect an existing service.
 
+<<<<<<< HEAD
```
Comment (sftim) [inline content/en/docs/concepts/services-networking/dual-stack.md:184]: This looks like a manual merge gone wrong; please fix it.

File: content/en/docs/concepts/services-networking/dual-stack.md [hunk 7]
```diff
@@ -230,6 +247,68 @@ dual-stack.)
...
+  - IPv4
+  ipFamilyPolicy: SingleStack
+  ports:
+  - port: 80
+    protocol: TCP
+    targetPort: 80
+  selector:
+    app.kubernetes.io/name: MyApp
+```
+>>>>>>> cca4c78633 ([en] update en docs to use recommended labels)
```
Comment (sftim) [inline content/en/docs/concepts/services-networking/dual-stack.md:311]: This looks like a manual merge gone wrong; please fix it.

**Review by sftim — CHANGES_REQUESTED** [review]
/remove-area blog

There are some merge conflicts that needed resolving manually but haven't been. @TheRealDwright, please fix those.

/hold
OK to unhold once the merge conflict byproducts are gone.

File: content/en/docs/concepts/services-networking/dual-stack.md (hunk 3 above)
Comment (TheRealDwright) [reply content/en/docs/concepts/services-networking/dual-stack.md]: Apologies, I must have missed this during one of the rebases I had to do.

File: content/en/docs/concepts/services-networking/dual-stack.md (hunk 4 above)
Comment (TheRealDwright) [reply content/en/docs/concepts/services-networking/dual-stack.md]: Apologies, I must have missed this during one of the rebases I had to do.

File: content/en/docs/concepts/services-networking/dual-stack.md (hunk 5 above)
Comment (TheRealDwright) [reply content/en/docs/concepts/services-networking/dual-stack.md]: Apologies, I must have missed this during one of the rebases I had to do.

File: content/en/docs/concepts/services-networking/dual-stack.md (hunk 6 above)
Comment (TheRealDwright) [reply content/en/docs/concepts/services-networking/dual-stack.md]: Apologies, I must have missed this during one of the rebases I had to do.

File: content/en/docs/concepts/services-networking/dual-stack.md (hunk 7 above)
Comment (TheRealDwright) [reply content/en/docs/concepts/services-networking/dual-stack.md]: Apologies, I must have missed this during one of the rebases I had to do.

**Comment by tengqm** [comment]
/hold cancel
Conflicts resolved.

**Comment by TheRealDwright** [comment]
@sftim is this something that there is any interest in merging?

**Review by Sea-n — COMMENTED** [review]
Thanks for your patient. The changes looks good to me now.
/lgtm

**Pull request merged**